src/server/routes/externalService.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2022       Ansh Goyal
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+
+import * as cbHelper from '../helpers/critiquebrainz';
+
+
+import express from 'express';
+
+
+const router = express.Router();
+
+router.get('/critiquebrainz/reviews', async (req, res) => {
```

Comment:
I'm not convinced by this URL structure.
I think compared to other routes, it would make more sense to have `/$entityType/$entityBBID/reviews`, which does mean adding that sub-route to each entity router.
And in that case we probably don't need to send the entity type in the query params.

What do you think? What was your thought process for this endpoint?

Reply:
Yes, I also think this URL structure is much cleaner and makes more sense. But we don't need to add a sub-route to each entity router. One main router can handle reviews of every entity type.